README.md

```diff
@@ -1,9 +1,127 @@
 # KubeSecondaryDNS
 DNS for KubeVirt VirtualMachines secondary interfaces
 
+## Prerequisites
+1. The KubeSecondaryDNS Deployment should be reachable from outside the cluster.
+2. The guest must have a guest agent installed, so the IP of the secondary interface will be reported.
+
+When running KubeSecondaryDNS on top on kubevirtci, we achieve [1] as follows:
+1. Create a NodePort for the Deployment, use `31111` as the port.
+2. Kubevirtci forwards all traffic from host `127.0.0.1:31111` to the kubevirtci container.
+3. Traffic in the container, on port `31111` is forwarded to the control plane node of the cluster.
+
+This way `nslookup -port=31111 <FQDN> 127.0.0.1` from outside the cluster can reach the secondary-dns container.
+
+## Deployment
+Choose the desired release from  
+https://github.com/kubevirt/kubesecondarydns/releases
+and apply it's manifest.  
+For example:
+```shell
+KSD_RELEASE=v0.0.5
+URL=https://github.com/kubevirt/kubesecondarydns/releases/download/${KSD_RELEASE}/secondarydns.yaml
+kubectl apply -f ${URL}
+```
+
+To undeploy it, use:
+```shell
+kubectl delete -f ${URL}
+```
+
+## Parameters
+You can update those paramters in the manifest.
+
+`DOMAIN` (default: `""`) - Determines the suffix of the FQDN.  
+Can be used in order to differentiate between different clusters (by including the cluster name as part of it for example).  
+The FQDN would look as follows:  
+If `DOMAIN` is not empty `<interface_name>.<vm_name>.<namespace>.vm.<DOMAIN>`  
+If `DOMAIN` is empty `<interface_name>.<vm_name>.<namespace>.vm`  
+
+`NAME_SERVER_IP` (default: `""`) - TODO
```

Comment:
IP value that is defined and used in order to reach KubeSecondaryDNS server from outside the cluster.

Reply:
Thanks
Please rephrase it so it won't imply this IP is something we listen to, but focus about the NS and A records

Reply:
Lets rephrase it a bit, and add it on a follow PR, thanks

Reply:
The @ symbol in the line before is responsible to add the $ORIGIN value (=domain name) before each one of those two lines.
Actually, I think I miss there a nameserver prefix in the A line, I'll add it in a separate PR.
Thanks for noticing.

Reply:
Thanks, added ns to the A line